lib/ruby_lsp/queue.rb

```diff
@@ -0,0 +1,186 @@
+# typed: strict
+# frozen_string_literal: true
+
+require "benchmark"
```

Comment:
What's this used for?

Reply:
We benchmark requests for telemetry, so that we can measure the server's performance.

Reply:
Right - it's just timing a block. I guess it seemed weird to me to see the word 'benchmark' in production code, but it's fine.